package.json

```diff
@@ -32,6 +32,7 @@
     "fix:helpers:cs": "black utils/ tests/",
     "install-dependencies": "npm run install-dependencies:client && npm run install-dependencies:server",
     "install-dependencies:client": "npm install --production=false --prefix client/",
+    "install-dependencies-ci:client": "npm ci --production=false --prefix client/",
```

Comment:
I think it would make sense to just use `npm ci` for `install-dependencies:client` – reading the [docs](https://docs.npmjs.com/cli/v10/commands/npm-ci), I do not see why one would want an unclean installation of dependencies.

Reply:
Indeed, fixed!